config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
-  root to: "pages#home"
+  devise_for :users
+  # root to: "pages#home"
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root to: "islands#index"
```

Comment:
Hey maybe the root path should be up top the file for better redabilaty, what do you think ? ❤️